modules/HomePage/components/TaskList/TaskList.tsx

```diff
@@ -185,21 +188,21 @@ const TaskList: React.FC<TaskListProps> = ({
               <Button
                 color="red"
                 style={{ display: task.completed ? "none" : "inline-block" }}
-                onClick={() => openModalHandler(task.id)}
+                onClick={() => openModalHandler(task.id ?? "")}
               >
                 Delete
               </Button>
               <Button
                 color="green"
                 style={{ display: task.completed ? "none" : "inline-block" }}
-                onClick={() => isCompleteHandler(task.id)}
+                onClick={() => isCompleteHandler(task.id ?? "")}
               >
                 Complete
               </Button>
               <Button
                 color="orange"
                 style={{ display: task.completed ? "none" : "inline-block" }}
-                onClick={() => startEditingTask(task.id)}
+                onClick={() => startEditingTask(task.id ?? "")}
```

Comment:
Why are we shortcircuiting for these values?

Reply:
This is my bad, forgot to remove these. I was trying to fix a type error I was receiving here. But the actual change would be part of the updateTodo PR as that is where I would work with editing task items. Will remove this change and make it part of the updateTodo PR.